app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC
+
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
```

Comment:
u don't need to inherit this class from ABC cause u have already inherit Player class from ABC